src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ CHANGELOG
    container parameter.
  * added ability to auto-refresh the toolbar if a `Symfony-Debug-Toolbar-Replace` header is set
    with any JavaScript truthy value
+ * addded a `Sfjs.loadToolbar(token)` function to the public `Sfjs` object so the toolbar can 
```

Comment:
usually we dont sell these features

Reply:
Should I remove? I'm curious how is this different than the header itself? Is the `Sfjs` object not considered "public" and therefore not subject to the BC promise etc?